ui/src/views/compute/CreateKubernetesCluster.vue

```diff
@@ -253,51 +198,31 @@
           </a-form-item>
           <div v-if="usePrivateRegistry">
             <a-form-item>
-              <span slot="label">
-                {{ $t('label.username') }}
-                <a-tooltip :title="apiParams.dockerregistryusername.description">
-                  <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
-                </a-tooltip>
-              </span>
+              <tooltip-label slot="label" :title="$t('label.dockerregistryusername')" :tooltip="apiParams.dockerregistryusername.description"/>
```

Comment:
`label.dockerregistryusername` Needs to be added to en.json

Reply:
^^ for following labels as well

Reply:
@davidjumani Reverted title to `label.username` as earlier